doc/hooks/versionCheckHook.section.md

```diff
@@ -16,20 +20,73 @@ stdenv.mkDerivation (finalAttrs: {
   nativeInstallCheckInputs = [
     versionCheckHook
   ];
+
   doInstallCheck = true;
 
   # ...
 })
 ```
 
-Note that for [`buildPythonPackage`](#buildpythonpackage-function) and [`buildPythonApplication`](#buildpythonapplication-function), `doInstallCheck` is enabled by default.
```

Comment:
This info about an otherwise external actor (namely the Python ecosystem) was inserted among two text blocks that describe the main character of this text, namely the versioncheckhook.

Please look at the solution proposed for this.